client/src/AppBundle/Controller/SettingsController.php

```diff
@@ -5,6 +5,7 @@
 use AppBundle\Entity as EntityDir;
 use AppBundle\Form as FormDir;
 use AppBundle\Service\Redirector;
+use Psr\Log\LoggerInterface;
```

Comment:
Doesn't look like we're using this anywhere in the class?

Reply:
Ah yes forgot to remove, I'll take it out